js/ui/date_box/ui.date_box.strategy.list.js

```diff
@@ -251,10 +266,20 @@ const ListStrategy = DateBoxStrategy.inherit({
 
     _dimensionChanged: function() {
         if(this._getPopup()) {
+            const popupWidth = this._getPopupWidth();
+
+            if(popupWidth === undefined) {
+                this.dateBox.setPopupOption('width', this._getInputWidth.bind(this));
+            }
+
             this._updatePopupHeight();
         }
     },
 
+    _getInputWidth() {
+        return this.dateBox.$element().outerWidth();
```

Comment:
```suggestion
        return this.dateBox._getInputWidth();
```
Perhaps it's make sense to rename `_getInputWidth` -> `getElementWidth` or `getWidth`?

Reply:
fixed